src/ray/raylet/lib/python/raylet_extension.cc

```diff
@@ -487,7 +487,7 @@ MOD_INIT(libraylet_library_python) {
   char common_error[] = "common.error";
   CommonError = PyErr_NewException(common_error, NULL, NULL);
   Py_INCREF(CommonError);
-  PyModule_AddObject(m, "common_error", CommonError);
+  PyModule_AddObject(m, "CommonError", CommonError);
```

Comment:
This comment doesn't need to be addressed in this PR. 
We can define this `CommonError`  in Python code and import it to the C extension. That would simplify the code. Also, the name `CommonError` sounds ambiguous to me. We should use more specific exception types depending on the concrete cases.

Reply:
I will temporarily change the name to RayCommonError. I think after @suquark 's cython change. This part will easier.